sql/src/main/java/io/crate/expression/scalar/string/StringPaddingFunction.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.expression.scalar.string;
+
+import io.crate.data.Input;
+import io.crate.expression.scalar.ScalarFunctionModule;
+import io.crate.expression.scalar.ThreeParametersFunction;
+import io.crate.metadata.BaseFunctionResolver;
+import io.crate.metadata.FunctionIdent;
+import io.crate.metadata.FunctionImplementation;
+import io.crate.metadata.FunctionInfo;
+import io.crate.metadata.Scalar;
+import io.crate.metadata.TransactionContext;
+import io.crate.metadata.functions.params.FuncParams;
+import io.crate.metadata.functions.params.Param;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+
+import java.util.List;
+import java.util.Locale;
+
+public class StringPaddingFunction extends Scalar<String, Object> {
+
+    public static final String LNAME = "lpad";
+    public static final String RNAME = "rpad";
+    public static final String DEFAULT_FILL = " ";
+    public static final int LEN_LIMIT = 1000;
```

Comment:
What is the reason for this limit?

Reply:
To avoid a user from trying 1000000000000000000000000000000000000000 which would result in crashing the node I suppose.

Reply:
can I leave this limit in place?, should I make it larger?, should I remove it altogether?

Reply:
I guess it makes sense to keep a limit, but it could probably be larger, maybe around 10000-50000?

Reply:
done!